### ISSUE TITLE
Would you mind if I use your script and continue it into it's own separate software?

### ISSUE THREAD

**Issue by CoderGhostG1** [Peekobot/peekobot#19]
I love your script and wish to create a separate business/code model based on it but I would like to ask for your permission first

I am not looking to make a quick buck but rather to continue the dev of this project into a more business friendly model for customers to use in their site as a plugin-Bot type of thingy.

**Comment by rosswintle** [comment]
Hi!

Thanks for this. The code is MIT licensed so you can do what you want with it, and I encourage you to do so. However, I'm really grateful that you asked permission.

I would draw attention to the part of the license that says:

> "The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software."

So your inclusion of MIT-licensed code must include the MIT license and allow others to base work from it. (This only applies to the Peekobot code, not to anything you build around it). If this is an issue then a competent JavaScript/front-end developer should be able to re-create a config-driven chat bot with little effort.

I would also VERY much appreciate it if you would not use the name "Peekobot". And I'd also love to know what you make with it. So feel free to pop back and let me know how you get on.

Thanks again - and all the best with the chatbot venture!

**Comment by CoderGhostG1** [comment]
Thanks for replying so fast :) 
I work mainly with PHP(Backend and Database) but have been looking to do other projects with other lang's so I have mainly been looking at vanilla JavaScript I wanted to make my own bot but thought I'll check GitHub first and see if there are any interesting idea's which I am glad I did, Your script seems so minimal and clean that it will be so easy to maintain on larger scale. 

My Idea: 
My current country does not have a lot of bots in the lang we use so my view is to create a consumers dashboard which they can then edit the bot by adding custom JS functions from the dashboard or for non-coding consumers they can just add custom Q's and A's. 
I feel it will help drive my countries tech environment as a lot of current big websites don't have any bots and if they have they don't really help so much 

Also I own a moderately successful marketplace websites which I built with PHP and would like to integrate a simple yet effective bot.

If you are interested I would love to have your view on my idea and what you think of it.
